configure

```diff
@@ -1148,6 +1157,84 @@ EOT
       }
       dump_and_unlink($DF) if $opts{'dry-run'};
     }
+
+    if (!$buildtao && !defined $opts{'safety-profile'}) {
+      # Read the existing features.
```

Comment:
This comment and those below in this chunk can probably be removed (or revised?)